CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing
+
+We are open to, and grateful for, any contributions made by the community. By contributing to Redux Toolkit, you agree to abide by the [code of conduct](https://github.com/reduxjs/redux-toolkit/blob/master/CODE_OF_CONDUCT.md).
+
+## Reporting Issues and Asking Questions
+
+Before opening an issue, please search the [issue tracker](https://github.com/reduxjs/redux-toolkit/issues) to make sure your issue hasn't already been reported.
+
+Please ask any general and implementation specific questions on [Stack Overflow with a Redux tag](http://stackoverflow.com/questions/tagged/redux?sort=votes&pageSize=50) for support.
```

Comment:
Maybe a redux-toolkit tag - or both - instead? I know I've seen that used over there.